packages/node/src/services.ts

```diff
@@ -56,41 +61,56 @@ class FirebaseMessagingService implements IMessagingService {
     private readonly messagingServerKey: string
   ) {}
 
-  async send(respondingAddress: Address, msg: object) {
-    await this.firebase
+  send(respondingAddress: Address, msg: object) {
+    const sanitizedMsg = JSON.parse(JSON.stringify(msg));
+    this.firebase
```

Comment:
So I thought this returns a promise, and I awaited it because I wanted confirmation on the operation succeeding, and throwing a reject when the operation failed.

This seems like it'd ignore the reject if the operation fails?